mod_reforged/hooks/skills/racial/alp_racial.nut

```diff
@@ -1,4 +1,52 @@
 ::mods_hookExactClass("skills/racial/alp_racial", function(o) {
+	local create = o.create;
+	o.create()
+	{
+		create();
+		this.m.Name = "Race: Alp";
```

Comment:
Just having `Alp` as the name is enough in my opinion. No need to prefix `Race:` to it. Many games do it e.g. they'll have a property called "Orc" on all orcs or "Elf" on all elves.

Reply:
I realize that "Race" is not the most correct term and I'm open for some other prefix suggestion.
But just the name is not descriptive enough for this type of effect while this is a line in the status effect.

Reply:
This is a standard thing in many tactical combat games. E.g. look at [this picture from Age of Wonders](https://steamuserimages-a.akamaihd.net/ugc/419188961970450761/641CE5D574D9406304B35A2B4CEAFDCB5EFACB0A/?imw=637&imh=358&ima=fit&impolicy=Letterbox&imcolor=%23000000&letterbox=true). Notice the `Wood Elf` "perk".

I think it will look just fine if you have an effect/perk showing up as "Alp" on an Alp which shows that are the effects that this entity has because of being an Alp. Similarly an effect called "Orc" on orcs which you can mouse over to see it. Humans can have "Human". It just means that "this character is a _____".